src/attributes.jl

```diff
@@ -278,16 +278,18 @@ struct NumberOfVariables <: AbstractModelAttribute end
 """
     ListOfVariableIndices()
 
-A `Vector{VariableIndex}` indexing all variables present
-in the model (i.e., of length equal to the value of `NumberOfVariables()`).
+A `Vector{VariableIndex}` containing all variable indices present in the model
+(i.e., of length equal to the value of `NumberOfVariables()`), ordered by
+creation time.
```

Comment:
"in the order in which they were added" (or something similar)
"creation time" almost suggests that the solver should keep a timestamp